### PR TITLE
feat(site-header): New mvp component

### PR DESCRIPTION
**Pull request by frankforland**

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -1,6 +0,0 @@
-root = true
```
Comment (ulrichgreen) [inline .editorconfig:1]: Why remove this? Rather fix it to match our style

File: src/components/site-header/site-header.module.css [hunk 2]
```diff
@@ -0,0 +1,18 @@
+.flosSiteHeader {
```
Comment (ulrichgreen) [inline src/components/site-header/site-header.module.css:1]: .flos-site-header

File: src/components/site-header/site-header.module.css [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    z-index: 999;
+    padding: 10px var(--spacing-7);
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    height: 60px;
+    box-sizing: border-box;
+}
+
+.flosSiteHeaderActionMenu {
```
Comment (ulrichgreen) [inline src/components/site-header/site-header.module.css:14]: .flos-site-header-action-menu

File: src/components/site-header/site-header.module.css (hunk 2 above)
Comment (ulrichgreen) [reply src/components/site-header/site-header.module.css:1]: Ah nevermind these - will be awkward when using them - brain fart

**Review by ulrichgreen — APPROVED** [review]
Beautiful

File: src/components/site-header/site-header.module.css (hunk 2 above)
Comment (frankforland) [reply src/components/site-header/site-header.module.css:1]: Yeah we are using the standard way - camlecase

image

**Pull request merged**